src/brfw.h

```diff
@@ -15,22 +15,23 @@
 const char * FC pointExt( const char *pNam);
 const char * FC pointNamExt( const char *pNam);
 //---------------------------------------------------------------------------
+#ifdef BINRES  // TODO (MP) Windows io header was replaced for C-style file io fopen.fclose, etc. These changes were not tested.
```

Comment:
You don't need these in this file, since it isn't being included if BINRES is defined.